modules/userId/eids.md

```diff
@@ -49,6 +49,13 @@ userIdAsEids = [
         }]
     },
 
+    {
+        source: 'ftrack.com',
```

Comment:
this does not match what you implemented

Reply:
Updated